backend/app.ts

```diff
@@ -27,9 +27,14 @@ const THREE_DAYS = 1000 * 60 * 60 * 24 * 3
 const TWO_MINUTES = 1000 * 60 * 2
 
 app.use(morgan("dev"));
-app.use(cors());
+
+app.use(cors({
+    credentials: true,
+    origin: 'http://localhost:3000'
```

Comment:
do we know yet what this will have to be once we deploy the app? should this instead be an env var?